elide-core/src/main/java/com/yahoo/elide/graphql/Environment.java

```diff
@@ -59,6 +68,6 @@ public Environment(DataFetchingEnvironment environment) {
     }
 
     public boolean isRoot() {
-        return parentResource == null;
+        return parentResource == null && !(rawSource instanceof GraphQLContainer);
```

Comment:
Do we really need both of these checks?

Reply:
We can probably safely stick with the `instanceof`.